server.js

```diff
@@ -24,7 +16,41 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Find Netflix data')
+})
+
+// Search the whole data base
+app.get('/movies', (req, res) => {
+  res.json(netflixData)
+})
+
+// Search on country and type Movie or TV Show
+app.get('/country/:country', (req, res) => {
```

Comment:
I think for this to follow the rules of REST, since the data it returns is an array of movies, this logic should be in the `/movies` endpoint. Then you could use a query param to filter, like `/movies?country=sweden`.